src/components/cylc/GScan.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <div>
+    <div v-if="workflows && workflows.length > 0">
+      <div v-for="workflow in workflows" :key="workflow.id">
+        <v-list-item :to="`/workflows/${ workflow.name }`">
+          <v-list-item-action>
+            <!-- TODO: the workflows are always running now, but later we will need https://github.com/cylc/cylc-ui/issues/90 adn update the icon displayed here -->
+            <v-icon>mdi-play-circle</v-icon>
+          </v-list-item-action>
+          <v-list-item-title>
+            <v-layout align-center align-content-center wrap>
+              <v-flex grow>{{ workflow.name }}</v-flex>
+              <v-flex shrink ml-4>
+                <!-- task summary tooltips -->
```

Comment:
Nice.

I think we might want to factor out some of this functionality into a common component so all views can benefit from these features (e.g. tooltips).

I was thinking of something like `JobBox` which either holds:
* State totals (as in this case).
* A collection of jobs (which collapses down to state totals if there are too many jobs).

Food for later thought...

Reply:
@oliver-sanders - can you post an issue for this, if you think it's needed.